server/applicationPattern/applicationPattern/services/ElasticsearchService.js

```diff
@@ -265,6 +265,97 @@ class ElasticsearchService {
       });
     }
   }
+
+  /**
+   * Method for scrolling results above 10k records.
+   *
+   * @description This approach uses 'scroll' API from Elasticsearch. This method should NOT
+   * be used when underlying data is changeable. Returns an empty array in response object when
+   * from param is bigger than the total number of records.
+   *
+   * @param {Number} from index from which we want to read documents
+   * @param {Number} size how many documents do we want to read
+   * @param {Object} query ES query to be used to filter the documents
+   * @param {String} [uuid] optional, UUID of ES client
+   * @throws Error if from+size do not exceed 10k
+   * @returns {Promise<Object>} { response, took } 
+   */
+  async scroll(from, size, query, uuid) {
+    // check if we should be using this method
+    if ((from + size) < 10000) {
```

Comment:
`(from + size) < 10000` should be `(from + size) <= 10000`

Reply:
Done.